package.json

```
@@ -80,206 +77,17 @@
-          "group": "vscode-ansible@3"
-        },
-        {
-          "command": "vscode-ansible.ssh",
-          "when": "resourceLangId == ansible",
-          "group": "vscode-ansible@4"
-        }
-      ]
-    },
-    "grammars": [
-      {
-        "language": "ansible",
-        "scopeName": "source.ansible",
-        "path": "./syntaxes/Ansible.tmLanguage"
-      }
-    ],
-    "configuration": {
-      "title": "Ansible Extension Configuration",
-      "type": "object",
-      "properties": {
-        "ansible.credentialsFile": {
-          "type": "string",
-          "default": "",
-          "description": "Ansible credential file path"
-        },
-        "ansible.terminalInitCommand": {
-          "type": "string",
-          "default": "default",
-          "description": "Command to be run in terminal after it starts"
-        },
-        "ansible.credentialsConfigured": {
-          "type": "boolean",
-          "default": false
-        },
-        "ansible.cloudShellConfirmed": {
-          "type": "boolean",
-          "default": false
-        },
-        "ansible.hover": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable/Disable hover over on module name"
-        },
-        "ansible.validation": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable/Disable validation on yaml content"
-        },
-        "ansible.autocompletion": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable/Disable ansible autocompletion"
-        },
-        "ansible.reuseSSHTerminal": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable/Disable reusing SSH terminal session"
-        },
-        "ansible.customOptions": {
-          "type": "string",
-          "default": "",
-          "description": "Custom options to run ansible playbook. eg. -i inventoryfile -vvv"
-        },
-        "ansible.fileCopyConfig": {
-          "type": "array",
-          "default": "",
-          "description": "Config file copying to remote host, including weather copy file on save. E.g. [ { \"server\": \"host\", \"sourcePath\":\"/mypath/\", \"targetPath\": \"/destpath/\", \"copyOnSave\", \"true\" }]"
-        },
-        "ansible.dockerImage": {
-          "type": "string",
-          "default": "microsoft/ansible:latest",
-          "description": "Custom docker image when running playbook in Docker."
-        },
-        "ansible.useWSL": {
-          "type": "boolean",
-          "default": "",
-          "description": "Use local Windows Subsystem for Linux to run Ansible"
-        },
-        "ansible.azureRestSpec": {
-          "type": "string",
-          "default": "",
-          "description": "Location of Azure REST specification."
-        }
-      }
-    }
+    "menus": {}
   },
   "scripts": {
     "compile": "cd server && npm install && cd .. && tsc -p ./ && tsc -p server/",
```

```suggestion
    "compile": "npm install && tsc -p ./",
```

"server" folder is removed, avoid CI fail